### PR TITLE
Fail, learn, reset, and transform

### PR DESCRIPTION
**Pull request by wpschaub**
Early DRAFT of thoughts I am gathering around reflecting and adjusting to change. Thoughts?

### CODE REVIEW

File: content/fail-learn-reset-transform.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+>
+> **FAILURE** comes in many shapes and sizes. Here are a few of many definitions from the Merriam-Webster Dictionary:
+>
+> - lack of success
+> - a falling short
+> - a fracturing or giving way under stress
+>
+
+Personally, I prefer of **FAILURE** as an "unexpected outcome" of doing, trying, evolving, or experimenting.
```
Comment (MerlinDarkstar) [inline content/fail-learn-reset-transform.md]: suggest:
I prefer of **FAILURE**
to:
I prefer to think of **FAILURE**

**Review by MerlinDarkstar — DISMISSED** [review]
Great article - what makes humans different?  Our ability to adapt to change; the more we do it, the more comfortable we are with this disruption as an opportunity.

File: content/fail-learn-reset-transform.md (hunk 1 above)
Comment (wpschaub) [reply content/fail-learn-reset-transform.md]: Great feedback.

**Review by danielrbroderick — DISMISSED** [review]

**Review by MerlinDarkstar — DISMISSED** [review]

**Review by AndreKaminski — DISMISSED** [review]
Looks good, thank you

**Review by danielrbroderick — APPROVED** [review]

**Review by MerlinDarkstar — APPROVED** [review]

**Pull request merged**